tests/test_confidence.py

```diff
@@ -181,3 +182,18 @@ def test_confidence_2d_limits(data, pars):
     assert_allclose(max(cx.ravel()), 0.02)
     assert_allclose(min(cy.ravel()), -4.0)
     assert_allclose(max(cy.ravel()), 1.0e-6)
+
+
+def test_confidence_prob_func(data, pars):
+    """Test conf_interval with alternate prob_func."""
+    minimizer = lmfit.Minimizer(residual, pars, fcn_args=data)
+    out = minimizer.minimize(method='leastsq')
+
+    def my_f_compare(best_fit, new_fit):
+        nfree = best_fit.nfree
+        nfix = best_fit.nfree - new_fit.nfree
+        dchi = new_fit.chisqr / best_fit.chisqr - 1.0
+        return f.cdf(dchi * nfree / nfix, nfix, nfree)
+
+    lmfit.conf_interval(minimizer, out, prob_func=my_f_compare)
+
```

Comment:
My checker does give a violention if I don't have a newline. I think it was pylint.

Reply:
might have to do with different line-endings on Windows, I don't know. For sure also ```pylint``` doesn't like this here: ```test_confidence.py:199:0: C0305: Trailing newlines (trailing-newlines)```

Reply:
Checked it again, no error even with flake8. Can we just disable this check or is there a good reason for that?

Reply:
no, we have the ```flake8``` checks there for a reason - to keep a correct and consistent coding style and exactly to avoid discussion like this on PRs

Reply:
I mean to disable to associated rule, not the flake8 check. After reading the rule I am even more confused now:

> There should be one, and only one, blank line at the end of each file. This warning will occur when there are zero, two, or more than two blank lines.

Which is exactly what I see on my side.